### DISCUSSION TITLE
Tapo P100 changing IP

### DISCUSSION THREAD

**Discussion by L0bit0** [petretiandrea/home-assistant-tapo-p100#353] [Q&A] [answered]
Hi

I have some tapo P100, 2 of then are keeping changing IP almost every day, even if my router is not restarted.

any idea why? do you know how to limit this IP changing?

thansk

**Comment by yognspoon** [comment, marked as answer]
Hi,

You can fix this by setting up an address reservation in your router.  This will assign the same IP address to the P100 everytime.

It will look something like this.

![image](https://user-images.githubusercontent.com/61858930/215640163-7726f9c0-054f-43b3-9873-f1e88c26cb2c.png)

Good luck.
